src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -222,19 +262,46 @@ public float DualUpdate(float output, float label, float alpha, float invariant,
             return maxNumThreads >= 2 && Math.Abs(fullUpdate) > Threshold ? fullUpdate / maxNumThreads : fullUpdate;
         }
 
-        public Double DualLoss(float label, float dual)
+        public double DualLoss(float label, float dual)
         {
             if (label <= 0)
                 dual = -dual;
 
             // The dual variable is out of the feasible region [0, 1].
             if (dual < 0 || dual > 1)
-                return Double.NegativeInfinity;
+                return double.NegativeInfinity;
 
             return _margin * dual;
         }
     }
 
+    /// <summary>
+    /// A smooth version of the Hinge Loss function, commonly used in classification tasks.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// Let $L(f(\vec{x}), y) = 1 - yf(\vec{x})$, where $f(\vec{x})$ is the predicted score and $y \in \{-1, 1\}$ is the true label.
```

Comment:
>$L(f(\vec{x}), y) = 1 - yf(\vec{x})$ [](start = 12, length = 36)

Can you mention somewhere that this is the definition of the hinge loss? Maybe something like "Let $...$ denote the hinge loss".